media/css/pebbles/components/_footer.scss

```diff
@@ -5,33 +5,61 @@
 @import '../includes/lib';
 
 #colophon {
+    @include open-sans;
     @include font-size(14px);
-    background: #fff;
-    color: $color-text-primary;
+    background: #000;
+    color: #fff;
     line-height: 1.5;
```

Comment:
Hmm, this was the spacing in the mockup but I do agree it seems tall. I think we could maybe drop it to 1.3 without folks really noticing. Thoughts?